cockroachdb/CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+#Contributing
```

Comment:
```suggesting
# Contributing
```

So the markdown processor will correctly make it a header line.

Reply:
Thank you, my local markdown editor displayed this as a header line despite the missing space.